Formula/o/overtls.rb

```diff
@@ -21,6 +21,8 @@ class Overtls < Formula
 
   def install
     system "cargo", "install", *std_cargo_args
+    # binary name change per https://github.com/ShadowsocksR-Live/overtls/commit/2f013f5e9c1df58af5b29337dddeae20d6bafdd5
+    bin.install_symlink "overtls-bin" => "overtls"
```

Comment:
Might be better to skip the symlink unless users complain and ask for it.

We should also update the `test` block to not use the symlink.

Reply:
works for me, we did retain the compatibility for some time for some other formula before though